README.md

```diff
@@ -85,6 +86,23 @@ Check Docker docs [here](https://github.com/moscajs/aedes-cli#docker)
 
 - [Examples](./docs/Examples.md)
 
+## Clusters
+
+Aedes needs on disk dbs like MongoDB and Redis in order to work with clusters. Based on our tests and users reports the best performances/stability are reached when using [aedes-persistence-mongodb] paired with [mqemitter-redis].
+
+Other info:
+
+- [aedes-persistence-redis] have some memory leak. Check [this issue](https://github.com/moscajs/aedes-persistence-redis/issues/73#issuecomment-686306508)
+- [mqemitter-mongodb] could have leaks when used in clusters. Check [this issue](https://github.com/mcollina/mqemitter-mongodb/issues/24)
+- You should assign a __FIXED BROKER ID__ to your Aedes instances:
```

Comment:
This should not be needed.

Reply:
Actually persistences store docs using also brokerId as reference, are you sure about this? 

Redis example:
https://github.com/moscajs/aedes-persistence-redis/blob/master/persistence.js#L294
https://github.com/moscajs/aedes-persistence-redis/blob/master/persistence.js#L295
https://github.com/moscajs/aedes-persistence-redis/blob/master/persistence.js#L327
https://github.com/moscajs/aedes-persistence-redis/blob/master/persistence.js#L345

What will happen if this instance dies and a new one comes up with a different id? Will be there any leak?

Reply:
In theory no, but maybe there are bugs. It's also more expensive so better send the message. There is the whole heartbeath logic in aedes to deal with that.

Reply:
Removed